ddtrace/internal/constants.py

```diff
@@ -0,0 +1,9 @@
+from typing import FrozenSet
+
+
+PROPAGATION_STYLE_DATADOG = "datadog"
```

Comment:
nit: we could probably implement these as literal types to take advantage of exhaustive type checking but we'd probably need to introduce `typing_extensions` as a dependency 😢

Reply:
yeah, that is worth a discussion since we have a ton of constants. future improvement we can make.